producer/n1n2_data_handler.go

```diff
@@ -11,7 +11,6 @@ import (
 	"github.com/omec-project/nas"
 	"github.com/omec-project/openapi/Nsmf_PDUSession"
 	"github.com/omec-project/openapi/models"
-	"github.com/omec-project/pfcp/pfcpType"
 	"github.com/omec-project/smf/consumer"
 	smf_context "github.com/omec-project/smf/context"
```

Comment:
It looks like there is no other `context`, so `smf_context` can be removed, no?
```suggestion
	"github.com/omec-project/smf/context"
```

Reply:
Done